gno.land/pkg/gnoland/node_inmemory.go

```diff
@@ -51,7 +53,7 @@ func NewDefaultGenesisConfig(pk crypto.PubKey, chainid string) *bft.GenesisDoc {
 }
 
 func NewDefaultTMConfig(rootdir string) *tmcfg.Config {
-	return tmcfg.DefaultConfig().SetRootDir(rootdir)
+	return tmcfg.TestConfig().SetRootDir(rootdir)
```

Comment:
can you explain why you're using TestConfig, in a comment

Reply:
because otherwise ChainID will be empty, and there is now actual way to change ChainID without a config file a the moment